exercises/strings/string2.js

```diff
@@ -0,0 +1,14 @@
+// strings2.js
+// All string methods return a new string. They don't modify the original string.
```

Comment:
For stuff like this, I think it might be better to introduce them on the first exercise for the concept. What do you think?

Reply:
Swapped the exercises.